packages/server-wallet/src/wallet/multi-threaded-wallet/manager.ts

```diff
@@ -20,31 +20,34 @@ export class WorkerManager {
   private pool?: Pool<Worker>;
   private threadAmount: number;
   private logger: Logger;
-  constructor(walletConfig: ServerWalletConfig) {
+  constructor(walletConfig: ServerWalletConfig, onNewWorker: (worker: Worker) => void) {
```

Comment:
nit but it might be nice to add some tsdocs here

Reply:
62efe6c54c7af52228fdde6ca801898f697af016